cmd/release-controller/sync.go

```diff
@@ -72,6 +72,9 @@ func (c *Controller) sync(key queueKey) error {
 		}
 	}
 
+	// ensure additional validation tests, before rejected tags are removed.
+	c.syncTests(release)
```

Comment:
This should be at the end, should return an error, and should be something like `syncAdditionalTesting`.